### PR TITLE
Let the Ahwassa projectile burn straight through ASF without being blocked

### PR DESCRIPTION
**Pull request by Crotalus**
Rebase of #933 

With this change it won't be possible to block off the bomb with ASF, the projectile will fly straight through and deal some damage on the asf flying into it. The radius is small though so no death of the whole cloud.

### CODE REVIEW

**Comment by Zockyzock** [comment]
how much damage & how big is the radius?

**Comment by Crotalus** [comment]
Very tniy collision radius atm, 2
https://github.com/FAForever/fa/pull/1283/commits/28e99a2e49ad1007e4854f8c08a800d4bc208334#diff-119b734e31c5dd1a26a6b7f4a9cae14aR16

Same damage as a normal bomb, 11k, i.e. kills asf which flies into it.

Both values should probably be moved out to blueprint so they are easily modified.

**Comment by Zockyzock** [comment]
Yes, that would be great.

**Comment by Zockyzock** [comment]
Can you move the values to the blueprint and then make a pull request to the balance branch?

**Comment by aeoncleanse** [comment]
@Crotalus - We'd like this with bp values, retargeted at balance/3660 please

File: lua/sim/Projectile.lua [hunk 1]
```diff
@@ -156,10 +156,15 @@ Projectile = Class(moho.projectile_methods, Entity) {
     end,
 
     OnCollisionCheck = function(self, other)
+        local same_army = self:GetArmy() == other:GetArmy()
```
Comment (aeoncleanse) [inline lua/sim/Projectile.lua]: Errrm... Why are we doing things in Projectile.lua? Or Unit.lua for that matter? Everything we need to do this change can be done in the Ahwassa projectile...

Comment (aeoncleanse) [inline lua/sim/Projectile.lua]: Got it. This is universal collision logic to allow projectiles to properly check collisions for friendly colliding allowed.

**Comment by aeoncleanse** [comment]
Recreated as https://github.com/FAForever/fa/pull/1545 with some fixes.

**Pull request closed without merging**